### DISCUSSION TITLE
Amethyst 0.22.0 Beta

### DISCUSSION THREAD

**Discussion by ianyh** [ianyh/Amethyst#1703] [Announcements]
# What's New

* Restore throwing windows between spaces!

## What's Changed
* Revive window grab space shifting by @ianyh in https://github.com/ianyh/Amethyst/pull/1698
* Migrate from MASShortcut to KeyboardShortcuts by @ianyh in https://github.com/ianyh/Amethyst/pull/1549

**Full Changelog**: https://github.com/ianyh/Amethyst/compare/v0.21.2...v0.22.0b

<hr /><em>This discussion was created from the release <a href='https://github.com/ianyh/Amethyst/releases/tag/v0.22.0b'>Amethyst 0.22.0 Beta</a>.</em>

**Comment by jugutierrezzz** [comment]
Yeeeiii I'm literally not updating my computer anymore so apple can't brake ameshyst haha
